smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/RuntimeClientPlugin.java

```diff
@@ -211,14 +223,23 @@ private RuntimeClientPlugin(Builder builder) {
     /**
      * Gets a list of additional parameters to be supplied to the
      * plugin function. These parameters are to be supplied to plugin
-     * function as an options hash. The list is empty if
-     * there are no additional parameters.
+     * function as second argument. The map is empty if there are
+     * no additional parameters.
      *
-     * @return Returns the optionally present list of parameters.
-     * @see #getPluginFunction()
+     * @param model Model the operation belongs to.
+     * @param service Service the operation belongs to.
+     * @param operation Operation to test against.
+     * @return Returns the optionally present map of parameters.
```

Comment:
nit: a little more details of map key and value?

Reply:
Added in [`043774a` (#353)](https://github.com/awslabs/smithy-typescript/pull/353/commits/043774aeaf64a4f0c0fd6d540f5665c462f071dd)